setup/pos_supplierinfo_search/odoo/addons/pos_supplierinfo_search

```diff
@@ -0,0 +1 @@
+../../../../pos_supplierinfo_search
```

Comment:
Is this setup part useful in commits ? won't it be generated by oca-bot ?

regards

Reply:
It will be generated by ocabot if missing, but I need it to here so that I can deploy this PR using pip